### PR TITLE
Adding needed imports to make literallycanvas work as imported depencecy

### PR DESCRIPTION
**Pull request by jarlep**
We had some issues upgrading our project to React 16 and using literallycanvas. Added some imports and requires in the files that referred directly to React

### CODE REVIEW

File: src/core/LiterallyCanvas.coffee [hunk 1]
```diff
@@ -1,3 +1,4 @@
+React = require '../reactGUI/React-shim'
```
Comment (anx-ckreuzberger) [inline src/core/LiterallyCanvas.coffee]: Why would you need to import it in the core/LiterallyCanvas file? This part of the program should be independent of the UI.

Comment (jarlep) [reply src/core/LiterallyCanvas.coffee]: Good catch. Indeed not needed in src/core/LiterallyCanvas.coffee.
Did gulp + npm pack and tested it in my application.

**Comment by jarlep** [comment]
Found another place React import was needed. The eyedropper button didn't work as React was not included in the commonjs output in lib/js/optionsStyles/stroke-or-fill.js

**Comment by irskep** [comment]
Thanks! Sorry for not getting to this sooner.

**Comment by irskep** [comment]
BTW, if any of you wants commit access, just drop me an email. I'm happy to let this project be carried forward by people who are using it.

**Comment by miguelduarte42** [comment]
Any chance this could be released to NPM?

**Comment by irskep** [comment]
When I Get Around To It: The Literally Canvas Story

**Comment by miguelduarte42** [comment]
Guys, I'm having some troubles importing the module. I'm using Create React App, and here's what I've done:

- cloned this repo (master branch)
- ran `gulp`
- copied the resulting `lib` folder to my project under the name `literallycanvas`
- imported the `index.js` file from that folder

And I get the following error:

```
./src/services/literallycanvas/js/core/shapes.js
  Line 677:  'LC' is not defined  no-undef
```
If I import `LiterallyCanvas` from `core`, I get this error:
```
./src/services/literallycanvas/js/core/actions.js
  Line 96:  'lc' is not defined  no-undef
```

Here's my simple Component. What am I doing wrong?

```
import React, { Component } from "react";
import LC from "services/literallycanvas/js/index";

class Canvas extends Component {
  constructor(props) {
    super(props);
  }
  render() {
    console.log(LC);
    return <div/>;
  }
}

export default Canvas;
```

**Comment by tgus** [comment]
We too verified that these recent patches for React 16 revolve those issues and are looking forward for a new NPM version to be released.

**Comment by fabien-tanguy** [comment]
Hi Guys, 
Having the same issue with module import in create react app under React 16. Unfortunately this PR #487 does not solve the problem.
I'll try to fork it and get back to you if I find a sustainable solution.

**Comment by tgus** [comment]
I've cloned the repository and this (incomplete code fragment) works for me with React 16.1.1:

```
import {LiterallyCanvasReactComponent as LC} from 'literallycanvas';
import LiterallyCanvas from 'literallycanvas/lib/js/core/LiterallyCanvas';
import defaultOptions from 'literallycanvas/lib/js/core/defaultOptions';

class MyCanvas extends React.Component {
    constructor(props) {
        super(props);
        defaultOptions.imageSize.width = 600;
        defaultOptions.imageSize.height = 400;
        this.lc = new LiterallyCanvas(defaultOptions);
    }
    render() {
        return (
            <LC lc={this.lc}/>  
        );
    }
}
```

**Comment by irskep** [comment]
Does one of you just want to take over the npm permissions for this library? I'm willing to transfer to someone with more of an investment.

Also, why fork the repo if you won't submit a PR?

**Comment by anx-ckreuzberger** [comment]
I can take over if you want. Let's handle the details via e-mail though.

**Comment by tgus** [comment]
@irskep 
When can we expect a new NPM version published? The current code in master fixes issues with React 16 as I've mentioned earlier.

**Comment by irskep** [comment]
Released. As I've said before, if anyone contacts me about taking over release responsibility, I'll definitely consider passing the torch.

**Pull request merged**